src/common/fees/calcMinerFeePerByte.ts

```diff
@@ -19,8 +19,20 @@ export const calcMinerFeePerByte = (
   networkFeeOption?: NetworkFeeOption,
   customNetworkFee?: string
 ): string => {
-  const { standardFeeLowAmount, standardFeeHighAmount } = fees
-  const { highFee, lowFee, standardFeeHigh, standardFeeLow } = fees
+  const {
+    highFeeFudgeFactor = '1',
+    lowFeeFudgeFactor = '1',
+    standardFeeHighAmount,
+    standardFeeHighFudgeFactor = '1',
+    standardFeeLowAmount,
+    standardFeeLowFudgeFactor = '1'
+  } = fees
+  let { highFee, lowFee, standardFeeHigh, standardFeeLow } = fees
+
+  highFee = mul(highFee, highFeeFudgeFactor)
```

Comment:
Is it important for the fee values to be integers? If so these should be rounded

Reply:
fair point. I'll fix